packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -107,47 +95,47 @@ When a specific client/IP/device identifier combination exceeds either the 60 re
 
 > **Note:** For all orgs created after September 2020, this feature is automatically set to the **Enforce and log per client (recommended)** mode.
 
-To configure client-based rate limiting for existing orgs:
+To configure the client-based rate limit for existing orgs:
```

Comment:
I was commenting on each instance, but there are too many
In general I think we should be consistent in how we refer to these limits. I think we can talk about "(_a_ or _the_) client-based rate limit" or the "client-based rate limit framework" but when we're talking about its actions maybe it should be "client-based rate limiting" – does that make sense?

Reply:
@peteryaworsky-okta , yes, that makes sense. The use is as follows:
- client-based rate limit > where "limit" is used as a noun.
- client-based rate limiting > where "limiting" is used as an adjective, as a descriptor for "framework" as per your example: "client-based rate limiting framework"